packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3701,6 +3701,167 @@ void main() {
     );
   });
 
+  testWidgets(
+    'Can drag the left handle while the right handle remains off-screen',
+    (WidgetTester tester) async {
+      // Text is longer than textfield width.
+      const String testValue = 'aaaaaaaaaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbbbbbbbbb';
+      final TextEditingController controller = TextEditingController(text: testValue);
+      final ScrollController scrollController = ScrollController();
+
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: Center(
+            child: CupertinoTextField(
+              dragStartBehavior: DragStartBehavior.down,
+              controller: controller,
+              scrollController: scrollController,
+            ),
+          ),
+        ),
+      );
+
+      await tester.enterText(find.byType(CupertinoTextField), testValue);
+      await tester.pump();
+      await tester.pump(const Duration(milliseconds: 200));
```

Comment:
If you do decide to keep enterText and still need to pump, can you consolidate these?  Ideally if you could just use one `pumpAndSettle` that would be best.

Reply:
Removed!